### PR TITLE
Landing page

### PR DESCRIPTION
**Pull request by govardhanreddyc79**
pr-1

### CODE REVIEW

**Comment by saicharan-gouru** [comment]
-UI looks simple and cool.
-The code is well structured and folders are well structured
-Good use of context.

Suggestions:
-You can add preview link as well as some description while raising PR

**Review by saicharan-gouru — APPROVED** [review]
-UI looks simple and cool.
-The code is well structured and folders are well structured
-Good use of context.

Suggestions:
-You can add preview link as well as some description while raising PR

File: src/App.css [hunk 1]
```diff
@@ -1,51 +1,8 @@
-.App {
-  text-align: center;
-}
+@import url("https://ozone-ui.netlify.app/Docs/doc.css");
+@import url("https://the-sports-adda.netlify.app/index.css");
+@import url("https://ozone-ui.netlify.app/Components/Button/button.css");
```
Comment (arpit01923) [inline src/App.css:3]: can you tell me what are you doing here

File: src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,19 @@
+import "./App.css";
+import React from "react"
+import Home from "./pages/home/home";
```
Comment (arpit01923) [inline src/App.jsx:3]: later use barrel exports here as the pages increases

File: src/backend/db/categories.js [hunk 3]
```diff
@@ -1,27 +1,40 @@
 import { v4 as uuid } from "uuid";
+import {balls, bats,stumps,gloves} from "../../assets/images"
```
Comment (arpit01923) [inline src/backend/db/categories.js]: use absolute paths here to avoid ../../

File: src/components/newDealsCard.jsx [hunk 4]
```diff
@@ -0,0 +1,15 @@
+export const NewDealsCard = ({dealItem}) => {
+    const {dealName, imageSource, imageAlt, link, discount} = dealItem
+    console.log(imageSource)
+    return(
+        <div className="new-arrivals">
+        <img className ="deal-image" src={imageSource} alt={imageAlt}/>
+        <div>
+            <h3>Available at {discount}% OFF</h3>
+            <a className = "cat-item margin-top" href="./ProductListing/productListing.html" target="_blank">
```
Comment (arpit01923) [inline src/components/newDealsCard.jsx:9]: - no need to use _blank 
- may be use _self

File: src/pages/home/home.jsx [hunk 5]
```diff
@@ -0,0 +1,64 @@
+import { Navbar } from "../../components/navbar";
+import { CategoryCard } from "../../components/categoryCard";
+import axios from "axios";
+import { sportsadda } from "../../assets/images";
+import { useState, useEffect } from "react";
+import { newDeals } from "../../staticData";
+import { NewDealsCard } from "../../components/newDealsCard";
```
Comment (arpit01923) [inline src/pages/home/home.jsx]: use absolute paths to avoid ../../

File: src/pages/home/home.jsx [hunk 6]
```diff
@@ -0,0 +1,64 @@
...
+
+
+
+const Home = () => {
+
+    const [ categories, setCategories] = useState([]);
+
+    useEffect(() => {
+        (async function () {
```
Comment (arpit01923) [inline src/pages/home/home.jsx:16]: if you are using es6 than maintain consistency

**Review by arpit01923 — APPROVED** [review]
Impressive
- clean UI
- good folder structure

What to change
- use absolute paths
- use barrel exports
- add description of the PR

File: src/pages/home/home.jsx (hunk 6 above)
Comment (govardhanreddyc79) [reply src/pages/home/home.jsx:16]: sure

File: src/pages/home/home.jsx (hunk 5 above)
Comment (govardhanreddyc79) [reply src/pages/home/home.jsx]: okay, will do it

File: src/components/newDealsCard.jsx (hunk 4 above)
Comment (govardhanreddyc79) [reply src/components/newDealsCard.jsx:9]: okay

File: src/App.css (hunk 1 above)
Comment (govardhanreddyc79) [reply src/App.css:3]: importing my CL css and Ecomm P0 screens css

File: src/App.jsx (hunk 2 above)
Comment (govardhanreddyc79) [reply src/App.jsx:3]: Done

File: src/backend/db/categories.js (hunk 3 above)
Comment (govardhanreddyc79) [reply src/backend/db/categories.js]: Sure

**Pull request merged**